data/dictionary/values getter by keys/83ac4d5d-9ac9-48d0-ae17-a5a77b18cfb6.json

```diff
@@ -18,6 +18,11 @@
         "type": "typeof`dict`[string][]",
         "name": "values",
         "description": "Sends the values corresponding to the received keys.\n\nExample:\n[1, 3]"
+      },
+      "47906a4f-9a5c-422f-a88f-f8b58331cade": {
+        "type": "string[]",
+        "name": "not found keys",
+        "description": "Sends the no found keys.\n\nExample:\n['second']"
```

Comment:
typo: "no**t** found keys"

Reply:
I'd use double quotes around `second` as the output is JSON and single quotes are not valid in JSON

Reply:
Done